src/NewTools-Compression-Utils/InflateStream.extension.st

```diff
@@ -0,0 +1,26 @@
+Extension { #name : 'InflateStream' }
+
+{ #category : '*NewTools-Compression-Utils' }
```

Comment:
Now that this method was moved here, is there a "sister" PR in the Pharo repository removing the extension from there?

Reply:
Ok, if there's no rush I will check it later today since now I'm about to start a meeting

Reply:
Sure, it's not for yesterday :)